crates/factors/src/lib.rs

```diff
@@ -17,9 +18,6 @@ pub use crate::{
 };
 
 // Temporary wrappers while refactoring
```

Comment:
I think this is no longer needed, right? The `Result` type is likely to stay I would think